generators/app/templates/Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:2.2.300 AS build-env
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS build-env
```

Comment:
Zijn de issues (DNS, memory leaks) die Alpine had voor 2.x opgelost in deze versie?

Reply:
Ok, dan zal ik deze bekijken en vervolgens de nodige aanpassingen doorvoeren